### PR TITLE
Add Event hooks docs in console

### PR DESCRIPTION
**Pull request by fakela**

### CODE REVIEW

File: docs/getting_started/event_hooks.md [hunk 1]
```diff
@@ -2,32 +2,29 @@
 title: Event Hooks
 sidebar_position: 9
 ---
+Event hooks service will receive all events from near-lake stream. It will then check defined webhooks and based on filters either call the destination service or not. Destination service is the user defined service whose url has a user set while creating a webhook. That service may have an open endpoint or require an auth token for access.
```
Comment (isoldo) [inline docs/getting_started/event_hooks.md:5]:
```suggestion
Event hooks service will receive all events from the near-lake stream.
It will then check the defined webhooks and based on filters either call the destination service or not. Destination service is the user defined service whose URL is set when creating a webhook.
That service may have an open endpoint or require an auth token for access.
```

File: docs/getting_started/event_hooks.md [hunk 2]
```diff
@@ -2,32 +2,29 @@
...
+User can specify with a webhook authorization mechanism so the service can authorise incoming data.
```
Comment (isoldo) [inline docs/getting_started/event_hooks.md:6]:
```suggestion
User can add an authorization token when creating a webhook.
```

File: sidebars.js [hunk 3]
```diff
@@ -77,18 +77,6 @@ const sidebars = {
         },
       ],
     },
-    {
```
Comment (isoldo) [inline sidebars.js:80]: Revert this change

File: docs/getting_started/event_hooks.md
Comment (isoldo) [inline docs/getting_started/event_hooks.md:1]: Move this file to a separate directory

**Review by isoldo — CHANGES_REQUESTED** [review]

**Review by chefsale — APPROVED** [review]
